### DISCUSSION TITLE
HA Energy Dashboard - Total or Daily?

### DISCUSSION THREAD

**Discussion by chunkysteveo** [klaasnicolaas/home-assistant-glow#166] [Q&A] [answered]
I've been running an older version fine with the "Total Energy" in the dashboard, and have just updated the ESPHome code to the V3.0.0 with the extra "Daily Energy". So with the Energy Dashboard for the grid - do I keep using "Total Energy" sensor, or change it to "Daily Energy" sensor, to track my grid usage? It looks like HA did it's own job of resetting "Total Energy" every day(?), but do I swap to "Daily" or just leave it? Are there advantages or disadvantages with either sensor?

Thanks - Steve

**Comment by klaasnicolaas** [comment, marked as answer]
You can just keep using the total energy entity, the day energy entity is something extra because some people like to have it.

**Reply by chunkysteveo** [reply]
Thanks for the update. I realised I lost my history by swapping the sensor in the energy dashboard so quickly set it back to Total.

Good to know though, thanks!
